conceptnet5/api.py

```diff
@@ -178,6 +177,26 @@ def lookup_single_assertion(uri):
         return success(response)
 
 
+def query_similarity(uri1, uri2):
+    """
+    Query the similarity between the two terms, uri1 and uri2. Return the
+    cosine similarity between the vectors of these two terms.
+    """
+    url = make_query_url('/similarity', [('uri1', uri1), ('uri2', uri2)])
```

Comment:
Can we call these "node1" and "node2"? The term "uri" doesn't play that big a role in the API anymore, and describes more than nodes anyway.